### ISSUE TITLE
no rating_average

### ISSUE THREAD

**Issue by badnaam** [edgarjs/ajaxful-rating#30]
I impleneted ajaxful rating on a Store model with 3 dimensions. I am
caching rating average for each dimension as well as the overall
average rating. The individual dimension rates are being saved and
shown correctly (including average per dimension) and everything works fine except, the "rating_average"
does not update (still at 0) even after multiple users have updated it
for a specific record.

**Comment by TeuF** [comment]
Same problem for me.

**Comment by edgarjs** [comment]
Copied from the readme file:

To use caching with dimensions, make sure you have a cache column defined for each dimension you want cached.
So if you want to cache the `spelling` dimension, you’ll need to have a column called `rating_average_spelling` on the articles table.
If you use a custom cache column name, follow the pattern `cache_column_name_dimension_name` to add cache columns for dimensions.

**Comment by TeuF** [comment]
I have no problem to cache dimensions, but I think what's about here, is to have a new column with the average rate:

rating_average_a = 4
rating_average_b = 3
rating_average_c = 3

rating_average = 3,33

Should we do it on app side or is it possible with ajaxful-rating ?

Thx,

**Comment by edgarjs** [comment]
Oh so you want to have the average calculated with all dimensions? That's not possible in the plugin yet, it only calculates the average for each dimension.

**Comment by TeuF** [comment]
Yes that would be very useful. Is it on the "To Code" list ? ;)

**Comment by edgarjs** [comment]
I think you're the first person that request it. So I will add it to the ToDo list, or if you want to add that feature and send me a pull request that'd be great, I haven't had too much time lately.

**Comment by TeuF** [comment]
I just comment the condition in has_many :rates_without_dimension and it work :)
It would be cleaner with a condition "if dimension define then no :conditions => ...."

<pre>
      if options[:dimensions].blank?
        has_many :rates_without_dimension, :as => :rateable, :class_name => 'Rate', :dependent => :destroy, :conditions => {:dimension => nil}
      else
        has_many :rates_without_dimension, :as => :rateable, :class_name => 'Rate', :dependent => :destroy
      end
</pre>


Then it update rating_average

**Comment by angela1111** [comment]
I agree a rating average would be quite useful!  Especially for my app!
